apps/events-helsinki/src/domain/search/eventSearch/utils.tsx

```diff
@@ -408,6 +419,18 @@ export const getOrganizationSearchUrl = (
   );
 };
 
+export const getHelsinkiOnlySearchUrl = (
+  source: EventFields | Venue,
+  router: NextRouter,
+  locale: AppLanguage
+): string => {
+  return routerHelper.getLocalizedCmsItemUrl(
+    ROUTES.SEARCH,
+    { helsinkiOnly: 'true' },
```

Comment:
Why wouldn't you use the HELSINKI_ONLY constant here?

Reply:
Taken into account in ea2b90da2f5c23a2b3a06d4586b31e6e964c9caa